less/home-page.less

```diff
@@ -1,22 +1,63 @@
 .home {
+  @media @mobile {
+
+    align-items: center;
```

Comment:
Align-items is a flex box property, but I don't see that you applied flexbox anywhere.